models/site.rb

```diff
@@ -1711,12 +1711,22 @@ def to_rss
 
           site_change_events.each do |event|
             event_link = "https://neocities.org/site/#{username}?event_id=#{event.id.to_s}"
-            xml.item {
-              xml.title "#{title} has been updated."
-              xml.link event_link
-              xml.pubDate event.created_at.rfc822
-              xml.guid event_link
-            }
+            if event.profile_comment_id
+              xml.item {
+                xml.title "#{event.actioning_site.username} has left a comment."
+                xml.description event.profile_comment.message
```

Comment:
I'm not familiar enough with Sequel to know if these will cause additional SQL queries or if they'll use the joined values. The `select` in the `Event.news_feed_default_dataset` query makes me think that the joined values will not be used, so I'll have to look into how to use the existing values instead of re-querying.